miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1589,4 +1593,40 @@ public static StainDto asDto(Stain from) {
     dto.setName(from.getName());
     return dto;
   }
+
+  public static Run to(NotificationDto from) {
+    Run to = null;
+    if (from instanceof PacBioNotificationDto) {
+      to = new PacBioRun();
+      to = setPacBioRunValues((PacBioNotificationDto) from, (PacBioRun) to);
+      to = setCommonRunValues(from, to);
+    } else if (from instanceof IlluminaNotificationDto) {
+      to = new IlluminaRun();
+      to = setIlluminaRunValues((IlluminaNotificationDto) from, (IlluminaRun) to);
+      to = setCommonRunValues(from, to);
+    } else {
+      throw new IllegalArgumentException(
+          String.format("The argument with the type '%s' cannot be converted into a Run.", from.getClass().getName()));
+    }
+    return to;
+  }
+
+  private static Run setPacBioRunValues(PacBioNotificationDto from, PacBioRun to) {
+    to.setMovieDuration(from.getMovieDurationInSec());
+    return to;
+  }
+
+  private static Run setIlluminaRunValues(IlluminaNotificationDto from, IlluminaRun to) {
+    to.setPairedEnd(from.isPairedEndRun());
+    return to;
+  }
+
+  private static Run setCommonRunValues(NotificationDto from, Run to) {
+    to.setName(from.getRunName());
+    to.setFilePath(from.getSequencerFolderPath().toString());
+    to.setHealth(from.getHealthType());
+    to.setStartDate(Date.from(from.getStartDate().atStartOfDay(ZoneId.systemDefault()).toInstant()));
```

Comment:
Would be good to add as a LimsUtils method if mixing date types is going to become more common